tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -679,6 +679,16 @@ public static class CIVisibility
             /// Configuration key for enabling Impacted Tests Detection.
             /// </summary>
             public const string ImpactedTestsDetectionEnabled = "DD_CIVISIBILITY_IMPACTED_TESTS_DETECTION_ENABLED";
+
+            /// <summary>
+            /// Configuration key for enabling or disabling the Test Management feature.
+            /// </summary>
+            public const string TestManagementEnabled = "DD_TEST_MANAGEMENT_ENABLED";
```

Comment:
The lack of consistency in prefix makes my eye twitch here, but I know it's out of your hands 😂

Reply:
😭